cmd/xdpcap/filter.go

```diff
@@ -85,13 +89,28 @@ func newFilterWithMap(hookMap *ebpf.Map, opts filterOpts) (*filter, error) {
 		actions:  opts.actions,
 	}
 
-	for _, action := range opts.actions {
-		program, err := newProgram(opts.filter, action, perfMap)
+	multiBufferMode := false
```

Comment:
The function signature is `func newProgram(..., xdpFrags bool)`, can we keep things consistent and use something with "frags" instead of `multiBufferMode` here?